ml_metadata/tools/mlmd_bench/thread_runner.cc

```diff
@@ -86,15 +87,16 @@ tensorflow::Status ExecuteWorkload(const int64 work_items_start_index,
 }
 
 // Merges all the thread stats inside `thread_stats_list` into a workload stats
-// and reports the workload's performance according to that.
+// and reports the workload's performance through command line output. Also,
+// passes `workload_summary` for updating with the performance result.
 void MergeThreadStatsAndReport(const std::string workload_name,
-                               ThreadStats thread_stats_list[], int64 size) {
+                               ThreadStats thread_stats_list[], int64 size,
```

Comment:
previous PR is submitted earlier today, please sync to the head, and apply fixes (e.g., using vector instead of array here)

Reply:
Will do.